File: minwook/10주차/제일_작은_수_제거하기.java
import java.util.*;

class Solution {
    public int[] solution(int[] arr) {
        int[] a = {-1};
        int min = arr[0];
        List<Integer> arrList = new ArrayList<>();
        for(int n : arr){
            min = Math.min(n,min);
        }
        for(int i=0; i<arr.length; i++){
            if(min != arr[i])
                arrList.add(arr[i]);
        }
        int[] answer = arrList.stream().mapToInt(x -> x).toArray();
        if(answer.length == 0)
            return a;
    
        return answer;
    }
}

File: jeonghyeon/10week/제일 작은 수 제거하기.js
function solution(arr) {
    let answer = [];
    if(arr.length <= 1) { // 하나있을 경우 -1
        return [-1];
    } else {
        arr.splice(arr.indexOf(Math.min(...arr)), 1); // 제일 작은 수 자름
        return arr;
    }   
}